backend/internal/deployment/kubedeployer/core.go

```diff
@@ -25,10 +42,13 @@ func (n *Node) AssignNodeIP(ctx context.Context, gridClient deployer.TFPluginCli
 
 	ip, err := getIpForVm(ctx, gridClient, networkName, n.NodeID)
 	if err != nil {
+		span.RecordError(err)
```

Comment:
<img width="884" height="574" alt="Image" src="https://github.com/user-attachments/assets/65c2597a-0ece-4062-8d20-b016bd9b5b31" />

Seems that we need to SetStatus as well
https://opentelemetry.io/docs/languages/go/instrumentation/#record-errors